storage/tests/unit/test_bucket.py

```diff
@@ -742,9 +742,22 @@ def test_list_blobs_defaults(self):
         self.assertEqual(kw["path"], "/b/%s/o" % NAME)
         self.assertEqual(kw["query_params"], {"projection": "noAcl"})
 
+    def test_list_blobs_objects_defaults(self):
+        NAME = "Name"
+        connection = _Connection({"items": []})
+        client = _Client(connection)
+        bucket = self._make_one(client=client, name=NAME)
+        iterator = bucket.list_blob_objects()
+        blobs = list(iterator)
+        self.assertEqual(blobs, [])
+        kw, = connection._requested
+        self.assertEqual(kw["method"], "GET")
+        self.assertEqual(kw["path"], "/b/%s/o" % NAME)
+        self.assertEqual(kw["query_params"], {"projection": "noAcl"})
+
```

Comment:
Please move this method below `test_list_blobs_w_all_arguments_and_user_project`.  The principle here is that the testcase methods should follow the order of the methods they exeercise, to make them easy to find by inspection.